### DISCUSSION TITLE
How to hide names of rules?

### DISCUSSION THREAD

**Discussion by nik-rev** [eslint/eslint#18664] [Q&A] [answered]
Hi, I just wanted to ask if its possible to disable showing the name of each rule?
eg "@typescript-eslint/no-unused-vars" should be hidden and "tailwindcss/enforces-shorthand" should also be hidden ideally.

It's just a bit obstructive and I would rather just be able to see the message.

![image](https://github.com/mfussenegger/nvim-lint/assets/154856872/e071751b-8780-420c-9b19-1659b6074ce6)

Thanks!

**Comment by Rec0iL99** [comment, marked as answer]
Hi @nikitarevenco, this is coming from your editor's ESLint extension. This is something we cannot fix. Please contact the maintainers of the extension for more help.
